Shoko.Server/API/v3/Models/Shoko/WebUI.cs

```diff
@@ -217,6 +217,14 @@ public WebUISeriesFileSummary(SVR_AnimeSeries series, HashSet<EpisodeType> episo
                     if (file?.Media == null || location == null)
                         return null;
 
+                    return new { file, xref, location };
+                })
+                .Where(f => f != null);
+            var files = filesWithXrefAndLocation
+                .Select(fileWithEpisode =>
+                {
+                    var (file, xref, location) = (fileWithEpisode.file, fileWithEpisode.xref, fileWithEpisode.location);
```

Comment:
you can use a tuplenin the .Select() above to get deconstruction working here (again)

Reply:
```suggestion
                    var (file, xref, location) = tuple;
```